tutorials/algorithms/03_vqe_simulation_with_noise.ipynb

```diff
@@ -186,158 +229,117 @@
-      "\n",
-      "VQE on Aer qasm simulator (with noise): -1.80204\n",
-      "Delta from reference energy value is 0.05524\n"
+      "  Specific qubit errors: [('id', (0,)), ('id', (1,)), ('id', (2,)), ('id', (3,)), ('id', (4,)), ('sx', (0,)), ('sx', (1,)), ('sx', (2,)), ('sx', (3,)), ('sx', (4,)), ('x', (0,)), ('x', (1,)), ('x', (2,)), ('x', (3,)), ('x', (4,)), ('cx', (3, 4)), ('cx', (4, 3)), ('cx', (3, 1)), ('cx', (1, 3)), ('cx', (1, 2)), ('cx', (2, 1)), ('cx', (0, 1)), ('cx', (1, 0)), ('measure', (0,)), ('measure', (1,)), ('measure', (2,)), ('measure', (3,)), ('measure', (4,))]\n"
      ]
     }
    ],
    "source": [
-    "import os\n",
-    "from qiskit.providers.aer import QasmSimulator\n",
-    "from qiskit.providers.aer.noise import NoiseModel\n",
-    "from qiskit.test.mock import FakeVigo\n",
-    "device_backend = FakeVigo()\n",
+    "from qiskit_aer import QasmSimulator\n",
+    "from qiskit_aer.noise import NoiseModel\n",
+    "from qiskit.providers.fake_provider import FakeVigo\n",
     "\n",
-    "backend = Aer.get_backend('aer_simulator')\n",
-    "counts1 = []\n",
-    "values1 = []\n",
-    "noise_model = None\n",
+    "# fake providers contain data from real IBM Quantum devices stored in Qiskit Terra,\n",
+    "# and are useful for extracting realistic noise models.\n",
+    "device_backend = FakeVigo()\n",
     "device = QasmSimulator.from_backend(device_backend)\n",
+    "\n",
     "coupling_map = device.configuration().coupling_map\n",
     "noise_model = NoiseModel.from_backend(device)\n",
-    "basis_gates = noise_model.basis_gates\n",
-    "\n",
-    "print(noise_model)\n",
-    "print()\n",
-    "\n",
-    "algorithm_globals.random_seed = seed\n",
-    "qi = QuantumInstance(backend=backend, seed_simulator=seed, seed_transpiler=seed,\n",
-    "                     coupling_map=coupling_map, noise_model=noise_model,)\n",
-    "\n",
-    "def store_intermediate_result1(eval_count, parameters, mean, std):\n",
-    "    counts1.append(eval_count)\n",
-    "    values1.append(mean)\n",
     "\n",
-    "var_form = TwoLocal(rotation_blocks='ry', entanglement_blocks='cz')\n",
-    "spsa = SPSA(maxiter=iterations)\n",
-    "vqe = VQE(ansatz, optimizer=spsa, callback=store_intermediate_result1, quantum_instance=qi)\n",
-    "result1 = vqe.compute_minimum_eigenvalue(operator=H2_op)\n",
-    "print(f'VQE on Aer qasm simulator (with noise): {result1.eigenvalue.real:.5f}')\n",
-    "print(f'Delta from reference energy value is {(result1.eigenvalue.real - ref_value):.5f}')"
+    "print(noise_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the noise model is defined, we can train our VQE using a noisy Aer Estimator. Please note that this simulation will take longer than the noiseless one."
```

Comment:
I don't think I would use the term `train` - we just run VQE right in the presence of noise

Maybe: Once the noise model is defined, we can run VQE using an Aer Estimator where we pass the noise model to the underlying simulator using the backend_options dictionary.

Reply:
Yes, this was from the old tutorial, I was still a bit confused by it but assumed it was right (learning I shouldn't assume anymore 😅)